internal/controller/constants.go

```diff
@@ -22,21 +26,38 @@ const (
 	OLSAppRedisDeploymentName = "lightspeed-redis-server"
 	// APIKeyMountRoot is the directory hosting the API key file in the container
 	APIKeyMountRoot = "/etc/apikeys" // #nosec G101
+	// CredentialsMountRoot is the directory hosting the credential files in the container
+	CredentialsMountRoot = "/etc/credentials"
+	// OLSAppCertsMountRoot is the directory hosting the cert files in the container
+	OLSAppCertsMountRoot = "/etc/certs"
 	// LLMApiTokenFileName is the name of the file containing the API token to access LLM in the secret referenced by the OLSConfig
 	LLMApiTokenFileName = "apitoken"
+	// OLSPasswordFileName is the name of the file containing password for its infra
+	OLSPasswordFileName = "password"
 	// OLSConfigFilename is the name of the application server configuration file
 	OLSConfigFilename = "olsconfig.yaml"
+	// OLSRedisSecretKeyName is the name of the key holding redis server secret
+	OLSRedisSecretKeyName = "password"
 	// Image of the OLS application server
 	// todo: image vesion should synchronize with the release version of the lightspeed-service-api image.
 	OLSAppServerImageDefault = "quay.io/openshift/lightspeed-service-api:latest"
 	// Image of the OLS application redis server
 	OLSAppRedisServerImageDefault = "quay.io/openshift/lightspeed-service-redis:latest"
 	// OLSConfigHashKey is the key of the hash value of the OLSConfig configmap
 	OLSConfigHashKey = "hash/olsconfig"
+	// OLSRedisConfigHashKey is the key of the hash value of the OLS's redis config
+	OLSRedisConfigHashKey = "hash/olsredisconfig"
+	// OLSRedisSecretHashKey is the key of the hash value of OLS Redis secret
+	// #nosec G101
+	OLSRedisSecretHashKey = "hash/redis-secret"
 	// OLSAppServerServiceName is the name of the OLS application server service
 	OLSAppServerServiceName = "lightspeed-app-server"
 	// OLSAppRedisServiceName is the name of OLS application redis server service
 	OLSAppRedisServiceName = "lightspeed-redis-server"
+	// OLSAppRedisSecretName is the name of OLS application redis secret
+	OLSAppRedisSecretName = "lightspeed-redis-secret"
+	// OLSAppRedisCertsName is the name of the OLS application redis certs secret
+	OLSAppRedisCertsSecretName = "lightspeed-redis-certs"
```

Comment:
Here too , Do we need the OLSApp in the Variable names ?

Reply:
Updated as suggested.